repondeur/zam_repondeur/views/lectures.py

```diff
@@ -87,27 +87,27 @@ def __init__(self, request: Request) -> None:
         )
         if self.lecture is None:
             raise HTTPNotFound
+        self.amendements = DBSession.query(Amendement).filter(
```

Comment:
Rename to `amendements_query` for clarity ? I thought it was a list of results at first.

Reply:
👌